prow/cluster/jobs/all-periodics.yaml

```diff
@@ -218,3 +230,31 @@ periodics:
         - https://gcsweb.istio.io/gcs/istio-release/
     nodeSelector:
       testing: test-pool
+
+- cron: "15 8 * * *" # every day at 12:15 am PST
+  name: daily-performance-benchmark
+  branches: master
+  decorate: true
+  extra_refs:
+    - org: istio-releases
+      repo: pipeline
+      base_ref: master
+  annotations:
+    testgrid-dashboards: istio_release-pipeline
+    testgrid-alert-email: istio-oncall@googlegroups.com
+    testgrid-num-failures-to-alert: '1'
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+      - <<: *istio_container_with_kind
+        env:
+          - name: GIT_BRANCH
+            value: master
+          - name: RELEASE_BOT
+            value: foo
+        command:
+          - entrypoint
+          - scripts/get_tools_repo.sh
+          - scripts/run-test.sh
```

Comment:
this also depends on the artifacts. You can find the latest build at https://storage.googleapis.com/istio-build/dev/latest which points to the last merged commit here: https://gcsweb.istio.io/gcs/istio-build/dev/